napari/layers/shapes/_shapes_models/_tests/test_shapes_models.py

```diff
@@ -78,9 +80,10 @@ def test_rectangle_rotate():
 def test_nD_rectangle():
     """Test creating Shape with a random nD rectangle."""
     # Test a single four corner planar 3D rectangle
```

Comment:
```suggestion
    """Test creating Shape with a single four corner planar 3D rectangle."""
```
It's not random anymore